### ISSUE TITLE
Process: IS final release

### ISSUE THREAD

**Issue by tkoeppe** [cplusplus/draft#1572]
We should take some extra care to polish the final document that we send out for publication.

To do so, I recommend that we make final, ad-hoc adjustments in a separate branch that we never merge into master and that contain changes that we wouldn't normally make since they are very brittle.

Specifically, I have something like this in mind:

1. Fork off the IS release branch
2. Change page headers and footers to match the intended final configuration, switch out the cover sheet, add the `Foreword` page from n4141 (see C++ Editors folder on Google Drive).
3. Remove all pagebreak-like breaks from the document: `\newpage`, `\pagebreak`, `\clearpage`. (Are there any more?)
4. Leaf through the entire document to make sure that the overall presentation is OK (e.g. avoid problems like in #1571), but also paying particular attention to vertical details:
   * Floats are not too dense and not too far from their reference point. Most of our floats have "h" placement, so we don't expect them to build up. If there's pressure, add a `\clearpage`. Here-floats should probably not leak into the next section.
   * Check if the TOC has a clause heading on the last line of the page. Not sure how we could fix this; perhaps we can add some leading whitespace before the beginning of the TOC.
   * Check if synopses have page breaks in them that split multiline declarations. Synopses often offer opportunities for whitespace insertion or removal to move things around a bit.
    * Check if footnote are split across multiple pages.
    * Least of all, see if other page elements are unduly separated by an intervening page break: itemdecls, itemdescrs, examples. We may not have a lot of options there, but a judicious page break here or there may well improve things.

The review work can probably be executed in parallel, one clause at a time, since there is no vertical correlation across clause boundaries.

Pre-fork ideas (work that can be done on the master branch):
* Double check overfull and underfull boxes.
* Spellcheck?
* Check all logs for errors and warnings.
* Think about font substitutions.

**Comment by Eelis** [comment]
Would the fork also mark the point at which C++20 changes (such as xspace removal) can start being merged into master? Or is that point farther in the future?

**Comment by tkoeppe** [comment]
@Eelis: This issue is unrelated to the present. It will only become relevant once we're actually about to publish. At the moment we haven't even started the DIS ballot yet.

**Comment by Eelis** [comment]
The question wasn't about the present though. It was about the relation between the moment of the fork and the moment C++20 changes can start being merged into master.

**Comment by tkoeppe** [comment]
I don't know the details for that, and those details are not the point of this issue I'm afraid.

**Comment by jwakely** [comment]
Table 133 is not placed very well. It belongs to [re.traits] but appears two pages later in the middle of a class synopsis in [re.regex].

**Comment by jensmaurer** [comment]
[N4141](http://www.open-std.org/jtc1/sc22/wg21/prot/14882fdis/n4141.pdf) doesn't seem to have a Foreword, just a cover page.

"Remove all pagebreak-like breaks from the document": We don't have a lot of these; and the ones we have in back.tex seem for starting the various indexes on fresh pages. We don't want to undo these. There are two `\clearpage`s in utilities.tex that look suspicious, though.

**Comment by jensmaurer** [comment]
I'm actually wondering whether we can make `\itemdecl`s and `\itemdescr` stick together on a single page automatically, or at least not have a page break within a paragraph inside these.

**Comment by tkoeppe** [comment]
@jensmaurer: I know we don't have a lot of page breaks. I count four load-bearing ones at the moment, two clearpages in utilities.tex, and two newpages in lib-intro and locales. I just don't want to forget about those.

**Comment by jensmaurer** [comment]
@tkoeppe: Agreed; my comment was ambiguous. I do not want to tie itemdecls to itemdescr.

**Comment by tkoeppe** [comment]
Added a note to watch out for split footnotes.

@K-ballo noted that footnote 118 is split across pages 291, 292. That particular one could probably be fixed by compressing the example in 15.5p2 a bit.

The https://www.ctan.org/pkg/fnbreak package is apparently able to emit warnings for split footnotes.
